src/main/java/edu/tseidler/model/PlayerList.java

```diff
@@ -50,8 +53,12 @@ public void switchStarting() {
         if (player1.isFirst()) {
             player1.unsetFirst();
             player2.setFirst();
-            add(player2);
-            add(player1);
+            playersList.clear();
+            playersList.add(player2);
+            playersList.add(player1);
+        } else {
+            player1.setFirst();
```

Comment:
Collapse to one method.
It would be nice to pass playersList to it, since it have access to both players